yb-voyager/src/srcdb/postgres.go

```diff
@@ -1006,25 +1006,20 @@ func (pg *PostgreSQL) GetMissingExportSchemaPermissions() ([]string, error) {
 		return nil, fmt.Errorf("error checking schema usage permissions: %w", err)
```

Comment:
even here, ideally, we should be taking in the schemas provided by the user, and not do a blanket check for all schemas.

Reply:
We are using the schema list provided by the user in all the queries.